misaligned/misaligned.java

```diff
@@ -10,9 +15,48 @@ static int printColorMap() {
         }
         return i * j;
     }
-    public static void main(String[] args) { 
+
+
+    static void printexpectedColorMap() {
+
+        int i = 0, j = 0;
+        for(i = 0; i < 5; i++) {
+            for(j = 0; j < 5; j++) {
+                System.out.printf("%d | %10s | %10s\n", (i * 5 + j)+1, majorColors[i], minorColors[i]);
+            }
+        }
+    }
+
+    public static void main(String[] args) {
         int result = printColorMap();
         assert(result == 25);
+
+        // Create a stream to hold the actual output
+        ByteArrayOutputStream actualOutputStream = new ByteArrayOutputStream();
+        PrintStream actualPrintStream = new PrintStream(actualOutputStream);
+
+        // Create a stream to hold the expected output
+        ByteArrayOutputStream expectedOutputStream = new ByteArrayOutputStream();
+        PrintStream expectedPrintStream = new PrintStream(expectedOutputStream);
+
+        PrintStream oldPrintStream  = System.out;
+        System.setOut(actualPrintStream);
+        printColorMap();
+
+        System.out.flush();
+        System.setOut(expectedPrintStream);
+        printexpectedColorMap();
+
+        System.out.flush();
+        System.setOut(oldPrintStream);
+
+        System.out.println("actual: \n" + actualOutputStream.toString());
+        System.out.println("expected: \n" + expectedOutputStream.toString());
+
+        assert(actualOutputStream.toString()==expectedOutputStream.toString());

```

Comment:
The number 5 used in the pair number calculation would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work if major color or minor color length is other than 5. Please think over it and write test cases accordingly to cover such cases.